ansible/roles/ufw/tasks/proxy_ufw.yml

```diff
@@ -18,6 +17,15 @@
     proto: any
   when: is_monolith
 
+- name: allow only private ssh traffic
+  ufw:
+    rule: allow
+    direction: in
+    port: 22
+    proto: any
+    interface: '{{ ufw_private_interface }}'
+  when: not is_monolith|bool
```

Comment:
for future reference and to avoid others having the same nightmare i had `not is_monolith|bool` is necessary here because both `is_monolith` and `not is_monolith` evaluate to `False`, likely because of implicit type coercions regarding strings, but explicit coercion to a bool fixes it.